Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
So, remove `REPRAPWORLD_KEYPAD` here and below on line 119 add a default for `REPRAPWORLD_KEYPAD_MOVE_STEP`…
```cpp
  #if ENABLED(REPRAPWORLD_KEYPAD)
    #define NEWPANEL
    #if ENABLED(ULTIPANEL) && !defined(REPRAPWORLD_KEYPAD_MOVE_STEP)
      #define REPRAPWORLD_KEYPAD_MOVE_STEP 1.0
    #endif
  #endif
```